network-api/networkapi/settings.py

```diff
@@ -55,6 +55,7 @@
     HEROKU_BRANCH=(str, ''),
     HEROKU_PR_NUMBER=(str, ''),
     HEROKU_RELEASE_VERSION=(str, None),
+    INDEX_PAGE_CACHE_TIMEOUT=(int, 60*60*24),
```

Comment:
With cache invalidation added, this might actually be too generous, but it's an env var, so we can always play with making the timeout longer via heroku